notebook/static/bigupload/js/cors/jquery.postmessage-transport.js

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
We do not include javascript dependencies in our repo. We use bower to express those dependencies.